css/at-rules/font-face.json

```diff
@@ -463,7 +463,7 @@
         "font-stretch": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-face/font-stretch",
-            "spec_url": "https://drafts.csswg.org/css-fonts/#descdef-font-face-font-stretch",
+            "spec_url": "https://drafts.csswg.org/css-fonts/#font-prop-desc",
```

Comment:
Question on this change: Looking at https://drafts.csswg.org/css-fonts/#font-prop-desc the MDN info boxes appear nicely next to font-stretch, font-style, etc. Would this change remove that and have all boxes at the "4.4. Font property descriptors: the font-style, font-weight, and font-stretch descriptors" heading?

Reply:
> Question on this change: Looking at [drafts.csswg.org/css-fonts/#font-prop-desc](https://drafts.csswg.org/css-fonts/#font-prop-desc) the MDN info boxes appear nicely next to font-stretch, font-style, etc. Would this change remove that and have all boxes at the "4.4. Font property descriptors: the font-style, font-weight, and font-stretch descriptors" heading?

Yes, it would. So in this case it’s probably better to leave these three as-is.

The thing is that the *“These descriptors define the characteristics of a font face and are used in the process of matching styles to specific faces…”* paragraph which follows these *Name…For…Value…Initial* blocks usually *precedes* the blocks, in most cases. And those introductory paragraphs usually have useful information highly relevant to developers. So if we don’t link to the headings in those cases, then readers coming in will never even see those introductory paragraphs — not unless they think/know to scroll back vertically to find it.

But anyway, this instance is not one of those cases — since the introductory paragraph in this case follows the *Name…For…Value…Initial* block rather than following it.

Reply:
Thanks, I saw this in other PRs too, so I think I prefer consistency. Let's just merge this. We can always change this later. For now it probably makes sense to apply the same rules everywhere.